src/dcap_provider.cpp

```diff
@@ -118,24 +124,39 @@ static std::string get_env_variable(std::string env_variable)
 
 static std::string get_collateral_version()
 {
-    std::string collateral_version = get_env_variable(ENV_AZDCAP_COLLATERAL_VER);
-    if (!collateral_version.empty())
+    std::string collateral_version =
+        get_env_variable(ENV_AZDCAP_COLLATERAL_VER);
+
+    if (collateral_version.empty())
+    {
+        log(SGX_QL_LOG_WARNING,
+            "Using default collateral version '%s'.",
+            default_collateral_version.c_str());
+        return default_collateral_version;
+    }
+    else
     {
-        if (!collateral_version.compare("v1") && !collateral_version.compare("v2"))
+        if (!collateral_version.compare("v1") &&
+            !collateral_version.compare("v2"))
         {
             log(SGX_QL_LOG_ERROR,
-                "Value specified in environment variable '%s' is invalid. Acceptable values are empty, v1, or v2",
+                "Value specified in environment variable '%s' is invalid. "
+                "Acceptable values are empty, v1, or v2",
                 collateral_version.c_str(),
                 MAX_ENV_VAR_LENGTH);
 
-            return std::string();
+            log(SGX_QL_LOG_WARNING,
+                "Using default collateral version '%s'.",
+                default_collateral_version.c_str());
+            return default_collateral_version;
         }
+
         log(SGX_QL_LOG_INFO,
             "Using %s envvar for collateral version URL, set to '%s'.",
             ENV_AZDCAP_COLLATERAL_VER,
             collateral_version.c_str());
-    }
-    return collateral_version;
+        return collateral_version;
+    }    

```

Comment:
>     [](start = 5, length = 4)

nit: extra spaces

Reply:
This is a result of running the format rules against the file